assets/js/components/organisms/CandidateMatchCandidate.jsx

```diff
@@ -14,27 +15,40 @@ class CandidateMatchCandidate extends Component {
     }
   }
 
+  toggleCategories() {
+    this.setState({
+      showCategory: !this.state.showCategory
+    })
+  }
+
   render() {
     return (
       <Card style={{marginBottom: '1em'}}>
-        <CandidateMatchProfileBadge
-          candidateName={this.props.candidateName}
-          compositeMatchScore={this.props.compositeMatchScore} />
-
-        {
-          this.state.showCategory ?
-          this.props.categoryMatchScores.map((category, index) => {
-            return (
-              <CandidateMatchCategory
-                key={index}
-                categoryName={category.categoryName}
-                categoryMatch={category.categoryMatch}
-                questions={category.questions} />
-            )
-          })
-          :
-          null
-        }
+        <div style={{display: 'flex'}}>
+          <CandidateMatchProfileBadge
+            candidateName={this.props.candidateName}
+            compositeMatchScore={this.props.compositeMatchScore} />
```

Comment:
To get the profile badge to fill the space and push the dropdown button to the right, pass a style object in props here.  You'll need to modify CandidateMatchProfileBadge to accept external styles though.  https://github.com/Code4HR/okcandidate/blob/master/assets/js/components/atoms/CandidateMatchRating.jsx#L35-L47